### PR TITLE
Task: 2232 funman UI

### PR DESCRIPTION
**Pull request by Tom-Szendrey**
I have not updated the output section. Trying to keep small PRs + there is a fair amount of not just css that will need to be done

image

### CODE REVIEW

**Review by shawnyama — APPROVED** [review]

File: packages/client/hmi-client/src/workflow/ops/funman/tera-funman.vue [hunk 1]
```diff
@@ -177,6 +191,10 @@ const outputId = computed(() => {
 	return undefined;
 });
 
+async function toggleAdditonalOptions() {
+	showAdditionalOptions.value = !showAdditionalOptions.value;
+}
+
```
Comment (mwdchang) [inline packages/client/hmi-client/src/workflow/ops/funman/tera-funman.vue:197]: async not needed here

File: packages/client/hmi-client/src/workflow/ops/funman/tera-funman.vue [hunk 2]
```diff
@@ -16,58 +16,60 @@
...
+				<div class="button-column">
+					<label>Tolerance</label>
+					<InputNumber
+						mode="decimal"
+						:min-fraction-digits="0"
+						:max-fraction-digits="7"
+						v-model="tolerance"
+					/>
+				</div>
+				<Slider v-model="tolerance" :min="0" :max="1" :step="0.01" />
```
Comment (mwdchang) [inline packages/client/hmi-client/src/workflow/ops/funman/tera-funman.vue:63]: Your input box tolerance and slider tolerance have different constraints
- input box is basically unbounded, while
- slider is between 0 and 1

What is the actual restriction on funman?

Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/workflow/ops/funman/tera-funman.vue:63]: Very good catch, was assuming between 0 and 1 based off of chats but I will need to confirm and fix one of these two inputs.

Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/workflow/ops/funman/tera-funman.vue:65]: Confirmed it should be between 0 and 1 inclusive

File: packages/client/hmi-client/src/components/funman/tera-constraint-group-form.vue [hunk 3]
```diff
@@ -142,14 +142,16 @@ watch(
...
 	display: flex;
 	padding: 1rem 1rem 1rem 1.5rem;
 	flex-direction: column;
 	justify-content: center;
 	align-items: flex-start;
 	border-radius: 0.375rem;
 	background: #fff;
+	border: 1px solid rgba(0, 0, 0, 0.08);
 	/* Shadow/medium */
 	box-shadow: 0px 2px 4px -1px rgba(0, 0, 0, 0.06), 0px 4px 6px -1px rgba(0, 0, 0, 0.08);
```
Comment (YohannParis) [inline packages/client/hmi-client/src/components/funman/tera-constraint-group-form.vue:156]: We have CSS variables for that: https://github.com/DARPA-ASKEM/terarium/blob/main/packages/client/hmi-client/src/assets/css/theme/_variables.scss#L863-L950

File: packages/client/hmi-client/src/workflow/ops/funman/tera-funman.vue [hunk 4]
```diff
@@ -84,27 +88,37 @@
...
-		<div class="right-side">
+		<div class="container">
 			<tera-funman-output v-if="outputId" :fun-model-id="outputId" />
 			<div v-else>
 				<img src="@assets/svg/plants.svg" alt="" draggable="false" />
 				<h4>No Output</h4>
 			</div>
 		</div>
 	</div>
+	<div class="footer">
```
Comment (YohannParis) [inline packages/client/hmi-client/src/workflow/ops/funman/tera-funman.vue:113]: You can simply use a `<footer>`, that has better semantic.

File: packages/client/hmi-client/src/components/funman/tera-constraint-group-form.vue (hunk 3 above)
Comment (Tom-Szendrey) [reply packages/client/hmi-client/src/components/funman/tera-constraint-group-form.vue:156]: Thanks i missed this first go through of the variables.css i will make a new PR for this cleanup

**Pull request merged**